### PR TITLE
5338: SEP Phase I

### PR DESCRIPTION
**Pull request by psk86**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [5338](https://devops.dchbx.org/redmine/issues/5338) |
| App-Dev Road Map # | 1.3.6.1 |
| Start Date | 08/12/2016 |
| Due Date | 10/02/2016  |
| App-Dev Priority | UP-3 |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | Holly Coffey |

### CODE REVIEW

**Comment by psk86** [comment]
This deployment also requires seeding the database with  qualifying_life_event_kinds_seed.rb
- 'date_options_available' boolean has been added to qualifying_life_event_kinds_seed.rb, which distinguishes a QLE if it has choice date options or not. This is important in testing the functionality.

**Comment by Kevin-Wei** [comment]
👍

**Comment by jlhicks4925** [comment]
RSpec
Finished in 9 minutes 42 seconds (files took 9.21 seconds to load)
4387 examples, 0 failures, 81 pending

Cucumber
61 scenarios (61 passed)
1014 steps (1014 passed)
6m12.248s


tag: 1.9.33

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -223,6 +227,7 @@ $(document).ready(function () {
   $(document).on("focus", "[class~='date-picker']", function(e){
     dateMin = $(this).attr("data-date-min");
     dateMax = $(this).attr("data-date-max");
+		console.log("blahhhhhh");
```
Comment (Kevin-Wei) [inline app/assets/javascripts/application.js.erb]: @arasevic um...

File: app/assets/javascripts/forms.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // $(".date-picker").datepicker();
 
 $(".date-picker").on("change", function () {
+  console.log('picket loaded');
```
Comment (Kevin-Wei) [inline app/assets/javascripts/forms.js]: also @arasevic

**Comment by Kevin-Wei** [comment]
Now tracking on #482

**Comment by Kevin-Wei** [comment]
addressed via #482

**Pull request closed without merging**